.pipelines/singletenancy/aks-swift/e2e-step-template.yaml

```diff
@@ -44,6 +44,17 @@ steps:
     name: "installKubetest"
     displayName: "Set up Conformance Tests"
 
+  - ${{ if eq( parameters['testDropgz'], true) }}:
+      - script: |
+          echo "##vso[task.setvariable variable=dropgzVersion]$(make cni-dropgz-test-version)"
+        name: dropgzTestVersion
+        displayName: "Dropgz Test Version"
+  - ${{ else }}:
+      - script: |
+          echo "##vso[task.setvariable variable=dropgzVersion]$(make cni-dropgz-version)"
+        name: dropgzTestVersion
+        displayName: "Dropgz Version"
```

Comment:
Nice! I liked how I could confirm that it is running a certain script by looking at the display name on the ADO pipeline.